ui/tools/objectinspector/objectinspectorwidget.cpp

```diff
@@ -52,12 +53,14 @@ ObjectInspectorWidget::ObjectInspectorWidget(QWidget *parent)
     ui->objectPropertyWidget->setObjectBaseName(QStringLiteral("com.kdab.GammaRay.ObjectInspector"));
 
     auto model = ObjectBroker::model(QStringLiteral("com.kdab.GammaRay.ObjectInspectorTree"));
+    ClientDecorationIdentityProxyModel *clientModel = new ClientDecorationIdentityProxyModel(this);
+    clientModel->setSourceModel(model);
     ui->objectTreeView->header()->setObjectName("objectTreeViewHeader");
-    ui->objectTreeView->setModel(model);
+    ui->objectTreeView->setModel(clientModel);
     ui->objectTreeView->setContextMenuPolicy(Qt::CustomContextMenu);
     ui->objectTreeView->setDeferredResizeMode(0, QHeaderView::Stretch);
     ui->objectTreeView->setDeferredResizeMode(1, QHeaderView::Interactive);
-    new SearchLineController(ui->objectSearchLine, model);
+    new SearchLineController(ui->objectSearchLine, clientModel);
```

Comment:
Good catch, it allow me to find and fix existing issues at some places with this search line controller!
Done